system.py

```diff
@@ -378,20 +378,47 @@ def __downloadBuildTool(cls, toolName):
     """
     ret = True
 
+    result = cls.downloadFromGoogle('chromium-' + toolName, os.path.join(Settings.localBuildToolsPath,toolName + '.exe.sha1'), False, False)
+
+    if not result:
+      ret = False
+      cls.logger.error('Failed downloading ' + toolName)
+    
+    return ret
+
+  @classmethod
+  def downloadFromGoogle(cls, bucket, path, isDirectory = False, shouldRecurse = True):
```

Comment:
Please, move this method above private methods section. 
  #---------------------------------- Private methods --------------------------------------------